portal-impl/src/portal.properties

```diff
@@ -6492,9 +6492,9 @@
     feature.flag.LPS-158473=false
 
     #
-    # Env: LIFERAY_FEATURE_PERIOD_FLAG_PERIOD__UPPERCASEL__UPPERCASEP__UPPERCASES__MINUS__NUMBER1__NUMBER5__NUMBER8__NUMBER4__NUMBER8__NUMBER2_
+    # Env: LIFERAY_FEATURE_PERIOD_FLAG_PERIOD__UPPERCASEL__UPPERCASEP__UPPERCASES__MINUS__NUMBER1__NUMBER5__NUMBER8__NUMBER4__NUMBER7__NUMBER8_
     #
-    feature.flag.LPS-158482=false
+    feature.flag.LPS-158478=false
```

Comment:
Hi @murilostodolni,
I think this  `feature.flag.LPS-158478` was already removed on Master. Could you please check? Thanks!

Reply:
Hi @carolmariaabb 

I did check this a few minutes. I see that [LPS-158478](https://issues.liferay.com/browse/LPS-158473) is IN QA and the LPS to remove FF is open. I think that conflict occurred because had an ordening of the FFs. What do you think? Thanks

Reply:
This links goes to https://issues.liferay.com/browse/LPS-158473 and not to [LPS-158478](https://issues.liferay.com/browse/LPS-158478), which is already closed

Reply:
perfect @carolmariaabb, it's really wrong. I removed the FF `feature.flag.LPS-158478`. We will send manually to Brian? The sf not work because the `.d.ts`.

Reply:
Hi @murilostodolni,

Let's wait for ` ci:test:relevant` to run at least once. Thanks!